tests/data/test_dataset.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the :mod:`pennylane.data.Dataset` class and its functions.
```

Comment:
:class:`pennylane.data.Dataset` ?

Reply:
I think this is copied from the DefaultGaussian tests that say:
"Unit tests for the :mod:`pennylane.plugin.DefaultGaussian` device."

I'll change it to :class: here because I think it makes more sense. If somebody knows why it's :mod: for the DefaultGaussian device, let me know.